### DISCUSSION TITLE
CARLA Off Road Environments

### DISCUSSION THREAD

**Discussion by gbbyrd** [carla-simulator/carla#6210] [General]
Hello,

Does anyone have any off road environments for CARLA? Are there any standard, non urban environments? I am using CARLA for autonomous vehicle research and would like to build a model for navigation in an offroad setting.

Thanks!

Grayson

**Comment by mayureshb7** [comment]
I could not find any more details than this one https://github.com/lssac7778

**Comment by RGring** [comment]
I am also interested in off road operations! I found this blog post (https://carla.org/2023/04/21/avl-off-road-simulation/), which advertises a very similar setup to what I am looking for, i.e. vehicle on fields. However, I could not identify the promise of the post: "AVL agreed to contribute their CARLA code changes and technical documentation to the CARLA project.". Anyone knows where I can find it?

**Comment by JazKarit** [comment]
They recently released an off-road map with CARLA 0.10.0 [https://carla.org/2024/12/19/release-0.10.0/](https://carla.org/2024/12/19/release-0.10.0/)
